index.html

```diff
@@ -40,65 +30,182 @@
-      data-thumbnail="https://placekitten.com/500/281">
-    </div>
-
-    <p>Vimeo video with thumbnail and title from API</p>
-    <div
-      class="lazyframe"
-      data-src="https://vimeo.com/45915667"
-      data-vendor="vimeo">
-    </div>
-
-    <p>Vimeo video with custom thumbnail and title</p>
-    <div
-      class="lazyframe"
-      data-src="https://vimeo.com/183524061"
-      data-vendor="vimeo"
-      data-title="Custom title"
-      data-thumbnail="https://placekitten.com/500/281">
-    </div>
-
-    <p>Iframe with custom theme and title</p>
-    <div
-      class="lazyframe lazyframe--custom"
-      data-src="https://www.surveymonkey.com/r/?sm=%2bd%2bdWobuoz7irMV%2fpzKPcJmrKosuGWSraJtIrFQ4wBQ%3d"
-      data-title="Something completely different">
-    </div>
-
-    <p>Google maps iframe with custom thumbnail and title that gets executed when in view</p>
-    <div
-      class="lazyframe lazyframe--custom"
-      data-src="https://www.google.com/maps/embed/v1/place?q=place_id:ChIJ8fA1bTmyXEYRYm-tjaLruCI&key=AIzaSyDQxRfx3HQrE0_oTFI2WHzoiGL_CM0JJfQ"
-      data-title="Something completely different"
-      data-initinview="true">
-    </div>
-  </div>
+
+<body class="overflow-x-hidden font-sans antialiased text-gray-900 bg-white">
+
+  <main>
+
+    <!-- start hero section -->
+    <section class="relative px-4 pt-10 pb-10 text-center bg-blue-500 md:text-center text-white">
+      <div>
+        <h1 class="text-5xl font-bold leading-tight">Lazyframe</h1>
+        <p class="mt-4">Dependency-free library for lazyloading iframes</p>
+        <p class="mt-4"><a href="https://github.com/jmartsch/lazyframe">https://github.com/jmartsch/lazyframe</a></p>
+      </div>
+    </section>
+    <!-- end hero section -->
+
+    <!-- start how it works -->
+    <section class="px-4 py-16 text-center bg-gray-100">
+      <h2 class="text-2xl font-bold leading-tight lg:text-3xl">What does it do? It turns blue.</h2>
+      <p class="mt-5 mt-5 text-2xl text-center">Lazyframe creates a responsive placeholder for embedded content and only
+        requests it when the user
+        interacts with
+        it. This decreases the page load and idle time. Lazyframe comes with brand-like themes for YouTube, Vimeo and
+        Vine.</p>
+      <h2 class="text-2xl font-bold leading-tight lg:text-3xl mt-10">Why?</h2>
+      <p class="mt-5 mt-5 text-2xl text-center">Because the JavaScript loaded from the external providers is big and
+        takes much time to load. This
+        slows down your site, even if your visitors don't want to see your beautiful videos.
+      </p>
+      <p>For example here are the number of requests and filesizes of some well-known services.</p>
+      <ul class="list-unstyled mt-10">
+
+        <li>
+          YouTube – 11 requests ≈ 580kb
+        </li>
+        <li>
+          Google maps – 52 requests ≈ 580kb
+        </li>
+        <li>
+          Vimeo – 8 requests ≈ 145kb
+        </li>
+      </ul>
```

Comment:
Please remove this. It's already in the readme

Reply:
I know it is in the Readme, but if you point someone to the example page, I think it should be self explaining. This makes it easier for users that are not coming directly to the github repo, but to the example page instead.

However, you are free to remove this before merging.